core-api/src/main/java/com/sequenceiq/cloudbreak/validation/SubnetValidator.java

```diff
@@ -12,33 +12,46 @@
 
 public class SubnetValidator implements ConstraintValidator<ValidSubnet, String> {
 
+    private SubnetType subnetType;
+
     @Override
     public void initialize(ValidSubnet constraintAnnotation) {
+        this.subnetType = constraintAnnotation.value();
     }
 
     @Override
     public boolean isValid(String value, ConstraintValidatorContext context) {
-        if (value == null) {
+        if (value == null || "0.0.0.0/0".equals(value)) {
```

Comment:
I do not understand 0.0.0.0/0 here: if Rfc1918Compliant is set, then the code would return with true, even though 0.0.0.0/0 is not RFC_1918 compliant. Also, a null value should return false - but I guess there is a reason why it is true here.

Reply:
Ok, thanks.